### ISSUE TITLE
Update client to support multiple tomogram features

### ISSUE THREAD

**Issue by manasaV3** [chanzuckerberg/cryoet-data-portal#1091]
blocked by https://github.com/chanzuckerberg/cryoet-data-portal/issues/897

## Motivation
To make it easier for users to transform annotations from one alignment to another, we should support a utility in the client to make the transformation.

## Definition of Done

1. The client has classes for Alignment, PerSectionAlignmentParameters and the newly added fields for existing classes.
2. The client supports a method that can take in the following parameters and create a transformed annotation

|  param | data type  | is_required  | default   | notes  |
|---|---|---|---|---|
|  annotation id  | int  | true  |   |   |
|  output alignment id |  int  | true  |   |   |
|  shape  |  str  | false |  None |  if no shape is provided, the action should be performed for all annotations |
| format |  str  |  false  | None  |   if no file format is provided, the action should be performed for all annotations  |
| ouput path  |  str  |  false  |  None  |  defaults to current  working directory |
|   |   |   |   |   |

3. There is a command line utility that can be utilized without using python
4. The client is released to prod.
5. There is sufficient documentation for users to understand this method
6. Sample code snippets are shared out to frontend and design for both the python method and the command line utility.

## Tasks
1. The client includes the new classes for Alignment, PerSectionAlignmentParameters and the newly added fields for existing classes from graphql
2. Create a method in [Annotation class template](https://github.com/chanzuckerberg/cryoet-data-portal/blob/01a8b6732fac7e890e6f9efc3a202cec3a20f562/client/python/cryoet_data_portal/src/cryoet_data_portal/templates/Annotation.jinja2) to support the transformation utility method.

#### case 1:  provided alignment id does not apply to this annotation
Throw an exception with a detailed error message

#### case 2:  provided input parameters are not valid for the annotation
Throw an exception with a detailed error message

#### case 3:  If a materialized annotation already exists for that alignment and annotation file format
Pull down the annotation from s3

#### case 4: No materialized annotation exists for that alignment and annotation file format
Transform the annotation using the affine transformation matrix. Reach out to Utz/Anchi for the logic to be implemented here.

3. Create a command line utility that calls the method created above

## Relevant Resources 
https://github.com/chanzuckerberg/cryoet-data-portal/blob/01a8b6732fac7e890e6f9efc3a202cec3a20f562/client/python/cryoet_data_portal

https://docs.google.com/document/d/1InjJKrx78Sv44QguMXR0p1o9Y34oPnMOY9J3aj375ks/edit#heading=h.qo1r1sf48iuy

**Comment by manasaV3** [comment]
cc: @uermel  @anchi2c

**Comment by manasaV3** [comment]
This is being moved to P1, and will be addressed in Q4, along with standardized annotations. 

cc: @junxini

**Comment by manasaV3** [comment]
Support for adding a method for alignment has been deferred. A new ticket will be created to capture that work.
